numpy/tests/typing/fail/scalars.py

```diff
@@ -32,11 +32,16 @@
 td_64 = np.timedelta64(1, "h")
 
 dt_64 + dt_64  # E: Unsupported operand types
-
 td_64 - dt_64  # E: Unsupported operand types
-td_64 / dt_64  # E: No overload
 td_64 % 1  # E: Unsupported operand types
-td_64 % dt_64  # E: Unsupported operand types
```

Comment:
This `pass` / `fail` structure makes it hard to see where this test went. Is there now a `pass` file that contains this test?

Reply:
See https://github.com/numpy/numpy/pull/17214/files#r481239379.